packages/user/style/menu.css

```
@@ -0,0 +1,52 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+.jp-MenuBar-label {
+  margin-left: 25px;
+}
+
+.jp-MenuBar-anonymousIcon {
```

Don't hesitate to merge the definition if they must match to highlight their link if modified later:

```suggestion
.jp-MenuBar-anonymousIcon,
.jp-MenuBar-imageIcon {
```

Same for the inner elements